src/lib/pickles_types/plonk_types.ml

```diff
@@ -484,16 +419,14 @@ module Evals = struct
         ; foreign_field_mul_selector
         ; xor_selector
         ; rot_selector
+        ; lookup_aggregation
+        ; lookup_table
+        ; runtime_lookup_table
```

Comment:
I'm missing some of the context, but is it ok for `runtime_lookup_table` to be twice?

Reply:
This looks like an error, yes.

Reply:
Fixed